api/src/controllers/cards.controller.ts

```diff
@@ -30,10 +30,12 @@ export class CardsController {
     try {
       // Initialize our page to 1 and our limit to 12 (12 cards per page)
       let page: number = Number(req.query.page) || 1;
+
+      const limit: number = Number(req.query.limit) || 12;
```

Comment:
I think that you implemented the limits feature well. Maybe also add a test for it in the api cards test file? Something along these lines: 

```
it('Get cards and limit', async () => {
      const result = await request(app.getServer()).get(`${cardsRoute.path}?page=1&limit=12`);

      expect(result.status).toEqual(200);
      expect(result.body.data.length).toEqual(12);

  
    });
```